### DISCUSSION TITLE
Favorites search

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#65] [Q&A] [answered]
I use favorites to handle mountain passes. I've got a file that hundreds of them in my country, Catalunya. I can see it on the map easily, but sometimes I would like to find a certain pass. I think should be useful to have a search box, similar as the existing in POI category.

This thread is very close to [that one](https://github.com/devemux86/cruiser/discussions/45).

**Comment by devemux86** [comment]
@airald thanks for the suggestion!

It is in my implementation list and I am already working on a _new universal search_ for locations, favorites, poi, etc.

**Comment by Cer-o** [comment]
Country Cataluña, jajajajaja

**Reply by airald** [reply]
yes, the definition of country (at least in Catalan and other languages) is not just restricted to borders, can be applied to other aspects like landscape, language, culture, etc. (ie: Basque Country, Country of Wales...)

**Reply by ramondevic** [reply]
Yes, Catalan Country, of course 👍

**Reply by Cer-o** [reply]
Jajajajaja.
Sure, like the multicolored country of Maya the Bee, Peter Pan's Neverland or Listenburg.
Keep dreaming and making a fool of yourself.

**Reply by devemux86** [reply]
_Please everyone, let's keep the discussion on topic and about **Cruiser**._ :slightly_smiling_face:

**Comment by DolomitesTom** [comment]
The search function in POI could be improved: when entering a keyword in the search field it could automatically position to this area, otherwise you are scrolling through tons of different areas and seldomily find the right one.

<img src="https://user-images.githubusercontent.com/121371629/223947007-46655c2b-779b-4d7a-bf50-98976c795fdc.jpg" width="33%"/>

**Comment by devemux86** [comment]
Let's continue the discussion in the main topic of Cruiser search at https://github.com/devemux86/cruiser/discussions/74.

**Comment by devemux86** [comment, marked as answer]
Search in favorites is available in [Cruiser 3.0.13](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

For more details see:
- https://github.com/devemux86/cruiser/discussions/74#discussioncomment-5369847
